content/integrations/lambdatest-integration.md

```diff
@@ -0,0 +1,54 @@
+---
+title: LambdaTest integration
+description: How to integrate your workflows with LambdaTest using codemagic.yaml
+weight: 6
+---
+
+**LambdaTest** is a cloud-based mobile testing platform that provides the ability to test your applications on real mobile devices. LambdaTest can be used as part of your Codemagic CI/CD pipeline to test your applications.
+
+## Create a LambdaTest account
+
+Registering with LambdaTest is required in order to be able to get the username and access token. You can sign up for free [here](https://www.lambdatest.com/).
+
+## Configuring Real Time and App Automation with Codemagic
+
+**LambdaTest** offers two testing environments, **Real Time** and **App Automation**. Applications can be submitted to both testing environments through Codemagic using a cURL request. In order to configure them correctly, you will need two pieces of data: **username** and **access token**. They can be found in the **LambdaTest UI** with your account. 
+
+## Real Time and App Automation
+
+You can test your **.ipa** and **.apk** directly on real devices by submitting them to the **LambdaTest** environment via a cURL request:
+
+```
+-Scripts:
+    - name: Submitting app to LambdaTest:
+    script: |
+        curl --location --request POST 'https://manual-api.lambdatest.com/app/upload/realDevice' --header 'Authorization: Basic $LAMBDATEST' --form 'name="lambda1"' --form 'appFile=@"app/build/outputs/apk/release/app-release.apk"'
+```
+
+
+Base64 encoding the username and token [here](https://mixedanalytics.com/knowledge-base/api-connector-encode-credentials-to-base-64/) with the **username:accessToken** format is needed, then save the encoded string to Codemagic as an environment variable. So the steps would be as follow:
+
+1. Encode and save the username and access token into an environment variable.
+2. Run the scripts in YAML referencing that environment variable.
+
+**$LAMBDATEST** environment variable is a custom name and can be named to anything preferred. More info can be found [here](https://docs.codemagic.io/variables/environment-variable-groups/) about how to set up environment variable groups with Codemagic.
```

Comment:
The information about username and token and how to save and reference them in Codemagic should come in the previous section before you give the actual script that uses them.

Reply:
Would not it be better that we give the cURL and then explain it ? Otherwise, it could be confusing

Reply:
I guess both approaches have their merits, so up to you. :)

Reply:
I myself usually find it very easy when I see the end product / result and then its explanation underneath. So, I'd suggest keeping it this way ))